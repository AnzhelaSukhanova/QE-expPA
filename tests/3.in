(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (and (and (or (bvult y (bvmul x #b0110)) (= y (bvmul x #b0110))) (or (bvult y (bvmul x #b0011)) (= y (bvmul x #b0011)))) (or (bvult y (bvmul x #b0010)) (= y (bvmul x #b0010))))))
