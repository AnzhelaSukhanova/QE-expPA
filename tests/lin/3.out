(let (($e1 (bvult #b0010 z))) (let (($e2 (bvult y #b0010))) (let (($e3 (bvult z #b0010))) (let (($e4 (bvult y z))) (let (($e5 (bvult #b0010 y))) (let (($e6 (bvult z y))) (not (and (not (and (not $e4) (not $e3))) (not (and (not $e2) (not $e1))) (not (and (not $e5) (not $e4))) (not (and (not $e6) (not $e1))) (not (and (not $e6) (not $e2))) (not (and (not $e5) (not $e3)))))))))))
